qa/suites/rbd/cache/workloads/fio.yaml

```diff
@@ -0,0 +1,32 @@
+tasks:
+- install:
+    extra_packages: ['ceph-immutable-object-cache']
+- ceph:
+    conf:
+      osd:
+        osd shutdown pgref assert: true
```

Comment:
Nit: not sure where this came from but shouldn't be needed

Reply:
done.